src/Passable.js

```diff
@@ -4,6 +4,7 @@ import enforce from './enforce';
 import passRunner from './pass_runner';
 import { passableArgs, initResponseObject, initField, onFail, root, runtimeError } from 'Helpers';
 import { Errors } from 'Constants';
+import { version } from '../package.json';
```

Comment:
`Passable.js` is the server side only correct? (so package.json won't end up in client bundle by mistake)

Reply:
That's a good point, Lior. Because package.json is an object, the rest of the tree won't be shaked.
You'll have to get it in a different way. How about a prebuild script?
Here's my suggestion:

#### (new) make-version-file.js
```js
const { version } = require('./package.json');
writeFileSync('version.json', JSON.stringify({version}));
```

#### (add) package.json
```json
   ...
    "prebuild": "node ./make-version-file.js",
   ...
```

#### (add) .gitignore
```
...
version.json
```

#### (modify) src/Passable.js
```js
import { version } from '../version.json';
```